src/kernel/MomentumActuatorSrcElemKernel.C

```diff
@@ -30,60 +30,54 @@ MomentumActuatorSrcElemKernel<AlgTraits>::MomentumActuatorSrcElemKernel(
   const stk::mesh::BulkData& bulkData,
   const SolutionOptions& solnOpts,
   ElemDataRequests& dataPreReqs,
-  bool lumped)
-  : Kernel(),
-    ipNodeMap_(sierra::nalu::MasterElementRepo::get_volume_master_element(AlgTraits::topo_)->ipNodeMap())
+  bool lumpedMass)
+  : lumpedMass_(lumpedMass)
 {
   const stk::mesh::MetaData& metaData = bulkData.mesh_meta_data();
   actuator_source_ = get_field_ordinal(metaData, "actuator_source");
   actuator_source_lhs_ = get_field_ordinal(metaData, "actuator_source_lhs");
   coordinates_ = get_field_ordinal(metaData, solnOpts.get_coordinates_name());
 
-  MasterElement* meSCV = sierra::nalu::MasterElementRepo::get_volume_master_element(AlgTraits::topo_);
-
-  if ( lumped ) {
-    get_scv_shape_fn_data<AlgTraits>([&](double* ptr){meSCV->shifted_shape_fcn(ptr);}, v_shape_function_);
-  }
-  else {
-    get_scv_shape_fn_data<AlgTraits>([&](double* ptr){meSCV->shape_fcn(ptr);}, v_shape_function_);
-  }
+  meSCV_ = MasterElementRepo::get_volume_master_element<AlgTraits>();
 
   // add master elements
-  dataPreReqs.add_cvfem_volume_me(meSCV);
+  dataPreReqs.add_cvfem_volume_me(meSCV_);
 
   // fields and data
   dataPreReqs.add_coordinates_field(coordinates_, AlgTraits::nDim_, CURRENT_COORDINATES);
   dataPreReqs.add_gathered_nodal_field(actuator_source_, AlgTraits::nDim_);
   dataPreReqs.add_gathered_nodal_field(actuator_source_lhs_, AlgTraits::nDim_);
   dataPreReqs.add_master_element_call(SCV_VOLUME, CURRENT_COORDINATES);
+  dataPreReqs.add_master_element_call(SCV_SHAPE_FCN, CURRENT_COORDINATES);
```

Comment:
If `lumpedMass_ == true` then `SCV_SHAPE_FCN` is never used in `execute`, I recommend that we do this in the `else` section of `if (lumpedMass_)`

Reply:
Agreed. I may have missed this in other PRs I did then. I will go back and create a PR for those as well.